modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -85,6 +96,19 @@ curl -ks <inference_endpoint_url>/v2/models/<model_name>/infer -d \
 -H 'Authorization: Bearer <token>' <1>
 ----
 <1> You must add the `Authorization` header and specify a token value _only_ if you enabled token authorization when deploying the model.
+
+*vLLM ServingRuntime for KServe*
+[source]
+----
+curl -v https://<inference_endpoint_url>:443/v1/chat/completions -H \
+"Content-Type: application/json" -d '{ \
+"model": "<model_name>", \
```

Comment:
Note that this query should work regardless of the `"model"` argument, so we could get rid of it

Reply:
Removed in latest commit